test/markdown.jl

```diff
@@ -39,3 +39,17 @@ writemime(io::IO, m::MIME"text/plain", r::Reference) =
     print(io, "$(r.ref) (see Julia docs)")
 
 @test md"Behaves like $(ref(fft))" == md"Behaves like fft (see Julia docs)"
+
+# GH tables
+@test md"""a|b
+1|2""" == MD(Table(Any[Any[Any["a"],Any["b"]];Any[Any["1"],Any["2"]]], :r))
+
+table = md"""| a  |  b | c |
+             | :- | -: | - |
+             | d`gh`hg | hgh**jhj**ge | f |"""
+@test table == MD(Table(Any[Any[Any["a"],Any["b"],Any["c"]],
+                            Any[Any["d",Markdown.Code("","gh"),"hg"],
+                                Any["hgh",Markdown.Bold(Any["jhj"]),"ge"],
+                                Any["f"]]],
+                        [:l, :r, :r]))
```

Comment:
It would be nice to make the tests as human-readable as possible, e.g. you can write this as
```julia
MD(Table(Any[["a","b","c"],
             Any[["d",Code("gh"),"hg"],
                 ["hgh",Bold("jhj"),"ge"],
                 "f"]],
         [:l, :r, :r]))
```
(This actually doesn't work at the moment, which I think is due to a problem with `print_align`)